src/lib/Simplify.hs

```diff
@@ -737,10 +739,10 @@ simplifyOp op = case op of
     Left  a   -> return a
     Right op' -> emitOp op'
 
-pattern IdentityReconAbs :: Abs binder ReconstructAtom n
+pattern IdentityReconAbs :: Abs binder (ReconstructAtom SimpIR) n
```

Comment:
All the other reconstructions are CoreIR; why is this SimpIR?

Reply:
Honestly, it should probably be a different third thing. It's going to have features of both.